src/java/org/apache/cassandra/config/CassandraRelevantProperties.java

```diff
@@ -426,6 +426,7 @@ public enum CassandraRelevantProperties
     SAI_LATEST_VERSION("cassandra.sai.latest_version", "aa"),
     SAI_MAX_FROZEN_TERM_SIZE("cassandra.sai.max_frozen_term_size_kb", "5"),
     SAI_MAX_STRING_TERM_SIZE("cassandra.sai.max_string_term_size_kb", "1"),
+    SAI_MAX_VECTOR_TERM_SIZE("cassandra.sai.max_vector_term_size_kb", "16"),
```

Comment:
I think this property and the two other above (`MAX_STRING_TERM_SIZE` and `MAX_FROZEN_TERM_SIZE`) should use the new config standard where the config the properties don't need to use a fixed unit. Instead, the size unit can be expressed in the user-specified value:
```java
SAI_MAX_FROZEN_TERM_SIZE("cassandra.sai.max_frozen_term_size", "5KiB"),
SAI_MAX_STRING_TERM_SIZE("cassandra.sai.max_string_term_size", "1KiB"),
SAI_MAX_VECTOR_TERM_SIZE("cassandra.sai.max_vector_term_size", "16KiB"),
```
Then callers can use `SAI_MAX_VECTOR_TERM_SIZE.getSizeInBytes()` to get the value in bytes.